packages/gensx/src/commands/start.tsx

```diff
@@ -75,14 +75,28 @@ export const StartUI: React.FC<Props> = ({ file, options }) => {
       throw new Error("Could not find tsconfig.json");
     }
 
-    let tsconfig;
+    let tsconfig: ts.ParsedCommandLine;
     try {
       const tsconfigContent = readFileSync(tsconfigPath, "utf-8");
       tsconfig = ts.parseJsonConfigFileContent(
         JSON.parse(tsconfigContent),
         ts.sys,
         process.cwd(),
+        {
+          incremental: false,
+          noEmit: false,
+        },
       );
+      tsconfig.options.incremental = false;
+      tsconfig.options.noEmit = false;
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      tsconfig.raw.compilerOptions.incremental = false;
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      tsconfig.raw.compilerOptions.noEmit = false;
+      tsconfig.options.outDir = ".gensx/dist";
+      tsconfig.options.target = ts.ScriptTarget.ES2015;
+      tsconfig.options.module = ts.ModuleKind.NodeNext;
+      tsconfig.options.moduleResolution = ts.ModuleResolutionKind.NodeNext;
```

Comment:
Do you think this will lead to weird breaks or edge cases? 

For instance, does this require `.js` imports for files that are related to the gensx deployment, but potentially not for others in the project?

Reply:
Had an idea out on my bike ride. I'll just add type: module to the package.json, and see how that works. We have the commonjs compatibility examples to validate against.

Reply:
I did add `type: module` to my nextjs package.json. It didn't break the next build / deployment, but I'm not 100% clear on the downstream implications of doing so on behalf of the user.

Reply:
Oh we shouldn't make it a permanent change, just for the build. My understanding is that it just affects the output (changing it from commonjs to ESM).